bin/pulseaudio-equalizer.in

```diff
@@ -74,8 +74,10 @@ if [ -f "$CONFIG_NAME" ]; then
     index=$((index+1))
     current=$((current+1))
   done
+  # trim first space
+  read  -rd '' trimmed <<< "$RAWDATA1"
```

Comment:
There's no real clean way to trim spaces in bash, so I would like to go to the more readable option.

Can we do `trimmed=$(echo $RAWDATA1 | sed 's/ *$//g')` instead?

Reply:
I agree to go with more readable option. Sure, we can use sed.